app/src/main/res/values/strings.xml

```diff
@@ -4,136 +4,137 @@
-    <string name="add_dialog_login_description">This topic requires you to login. Please type in a username and password.</string>
+    <string name="add_dialog_login_description">Please type in your username and password.</string>
     <string name="add_dialog_login_username_hint">Username</string>
     <string name="add_dialog_login_password_hint">Password</string>
-    <string name="add_dialog_login_error_not_authorized">Login failed. User %1$s not authorized.</string>
+    <string name="add_dialog_login_error_not_authorized">Login failed. The user \"%1$s\" is not authorized.</string>
     <string name="add_dialog_login_new_user">New user</string>
 
     <!-- Detail activity -->
-    <string name="detail_deep_link_subscribed_toast_message">Subscribed to topic %1$s</string>
+    <string name="detail_deep_link_subscribed_toast_message">Subscribed to \"%1$s\" topic</string>
     <string name="detail_no_notifications_text">You haven\'t received any notifications for this topic yet.</string>
     <string name="detail_how_to_intro">To send notifications to this topic, simply PUT or POST to the topic URL.
     </string>
     <string name="detail_how_to_example"><![CDATA[ Example (using curl):<br/><tt>$ curl -d \"Hi\" %1$s</tt> ]]></string>
-    <string name="detail_how_to_link">For more detailed instructions, check out the ntfy.sh website and documentation.
+    <string name="detail_how_to_link">More detailed instructions on ntfy.sh and in the documentation.
     </string>
-    <string name="detail_clear_dialog_message">Do you really want to delete all of the notifications in this topic?
+    <string name="detail_clear_dialog_message">Delete all of the notifications in this topic?
     </string>
-    <string name="detail_clear_dialog_permanently_delete">Permanently delete</string>
+    <string name="detail_clear_dialog_permanently_delete">Delete for good</string>
     <string name="detail_clear_dialog_cancel">Cancel</string>
     <string name="detail_delete_dialog_message">
-        Do you really want to unsubscribe from this topic and delete all of the
-        notifications you received?
+        Unsubscribe from this topic and delete all
+        notifications you have ever received?
     </string>
-    <string name="detail_delete_dialog_permanently_delete">Permanently delete</string>
+    <string name="detail_delete_dialog_permanently_delete">Delete for good</string>
     <string name="detail_delete_dialog_cancel">Cancel</string>
-    <string name="detail_test_title">Test: You can set a title if you like</string>
-    <string name="detail_test_message">This is a test notification from the Ntfy Android app. It has a priority of %1$d.
-        If you send another one, it may look different.
+    <string name="detail_test_title">Test: You can set a title if you like.</string>
+    <string name="detail_test_message">This is a test notification from the ntfy Android app. It has a %1$d priority.
+        If you send another, it may look different.
```

Comment:
"It has a 4 priority" is just weird and wrong. Revert this?

Reply:
I thought the placeholder was "high" "low" or some-such.